koku/masu/processor/ocp/ocp_cloud_updater_base.py

```diff
@@ -117,10 +117,10 @@ def _generate_ocp_infra_map_from_sql_trino(self, start_date, end_date, check_cac
                     end_date,
                     gcp_provider_uuid=self._provider_uuid,
                 )
+        if infra_map is None:
```

Comment:
Could we not just make this
```suggestion
        if not infra_map:
```
and keep all the `return {}` in the ocp_report_db_accessor? Is there a point in caching a result that is `{}`?

Reply:
This is a very valid point Michael and one that I had not considered. Part of the sql changes I suggested to Luke was for us to switch us from trying to correlate with every possible OpenShift cluster to just those that are uncorrelated. I think once that optimization goes in, we probably just don't need this cache anymore. I will go ahead and update the PR to remove the cache.